linode/manager
# feat: [M3-7351] - AGLB Configurations Add Route Drawer and other refinements

## Description 📝

- Updates AGLB Configurations to match UX mockups
- Adds Add Route drawer where you can
  - Add an existing route
  - Create a new route
- Adds `RouteSelect` component
- Fixes some position/styling
- Componentizes the Beta Regions 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-10-30 at 6 14 21 PM](https://github.com/linode/manager/assets/115251059/a956b934-4fdc-421a-b8be-5a477f80446b) | ![Screenshot 2023-10-30 at 6 13 50 PM](https://github.com/linode/manager/assets/115251059/1aeda3f7-7d20-468f-9c63-943a03171704) |
| ![Screenshot 2023-10-31 at 9 58 41 AM](https://github.com/linode/manager/assets/115251059/96be9573-2795-4761-8504-b1ce15484701) | ![Screenshot 2023-10-31 at 9 58 26 AM](https://github.com/linode/manager/assets/115251059/b71810b8-056b-4c85-bc71-fc61505d3b7f) |

## How to test 🧪

### Prerequisites
- Turn on the MSW 

### Verification steps 
- Go to http://localhost:3000/loadbalancers/0/configurations
- Check the UI for styling issues
- Check functionality of the `Add Route` button and the corresponding drawer (We can only test so much with the MSW)

## As an Author I have considered 🤔
